tutorials/MD/03_MD_Setup_Production_Run.ipynb

```diff
@@ -0,0 +1,274 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Connect to the Advanced Research Computing (ARC)\n",
+    "\n",
+    "Open a terminal and remotely connect to ARCUS using ssh (secure shell), using your username:\n",
+    "\n",
+    "`% ssh -X username@oscgate.arc.ox.ac.uk`\n",
+    "\n",
+    "When prompted, enter your password and hit Enter.\n",
+    "Now, you need to connect to the arcus-htc cluster, so in your command line type:\n",
+    "\n",
+    "`% ssh -X arcus-htc`\n",
+    "\n",
+    "If you get a message like this one:\n",
```

Comment:
Let's remove these details from here and move them to 00_setup. This won't happen if they log in again and it may just lead to confusion.

Reply:
RSA bit removed. I kept the login instructions.